subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio.h

```diff
@@ -5,6 +5,48 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+/* Set of macros related with Radio packet configuration flags */
+#define RADIO_PKT_CONF_
+/* PDU type, 1 bit field*/
+#define RADIO_PKT_CONF_PDU_TYPE_POS (0U)
+#define RADIO_PKT_CONF_PDU_TYPE_MSK BIT(RADIO_PKT_CONF_PDU_TYPE_POS)
+#define RADIO_PKT_CONF_PDU_TYPE_AC (0U)
+#define RADIO_PKT_CONF_PDU_TYPE_DC (1U)
+/* PHY type, two bit field */
+#define RADIO_PKT_CONF_PHY_POS (1U)
+#define RADIO_PKT_CONF_PHY_MSK (BIT_MASK(2) << RADIO_PKT_CONF_PHY_POS)
```

Comment:
Should it have been
```suggestion
#define RADIO_PKT_CONF_PHY_MSK (BIT_MASK(2U) << RADIO_PKT_CONF_PHY_POS)
```
Now that all other values are suffixed with `U`?

Reply:
corrected